### PR TITLE
Malin's news site

### PR DESCRIPTION
**Pull request by malin798**
Assignment from Technigo bootcamp. Includes creating a basic webpage which is responsive and had different designs for mobile, desktop and tablet. 

I started with the basic layout of the page, I used flexbox to do so. I made a simple sketch to create an overview of the finished page. If I had more time to work on the page I would like to add some more animations (and some more fun pictures).

https://agitated-shirley-69f2c3.netlify.com

### CODE REVIEW

**Comment by ericaglimsholt** [comment]
Hi! Could you add your full name to this PR (which is private) or on your profile?

**Comment by malin798** [comment]
> Hi! Could you add your full name to this PR (which is private) or on your profile?

Added my name on my profile!

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,106 @@
...
+
+      </div>
+      
+      <div class="desktop-menu-navbar">
+
+      <span class="navbar-link"><a class="link-desktop-navbar" href="" alt="link" target="_blank">News</a></span>
+      <span class="navbar-link"><a class="link-desktop-navbar" href="" alt="link" target="_blank">Sports</a></span>
+      <div class="site-name-tablet">
+        <img src="hiclipart.com.png" class="img-logo" alt="company-logo"> 
+        <h1>Techtrends</h1>
```
Comment (ericaglimsholt) [inline code/index.html]: Try to use only one `<h1>` per page! :)

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,106 @@
...
+      <h1>Techtrends</h1>
+
+      </div>
+      
+      <div class="desktop-menu-navbar">
+
+      <span class="navbar-link"><a class="link-desktop-navbar" href="" alt="link" target="_blank">News</a></span>
+      <span class="navbar-link"><a class="link-desktop-navbar" href="" alt="link" target="_blank">Sports</a></span>
+      <div class="site-name-tablet">
+        <img src="hiclipart.com.png" class="img-logo" alt="company-logo">
```
Comment (ericaglimsholt) [inline code/index.html]: Alt text is written like "normal" text
```suggestion
        <img src="hiclipart.com.png" class="img-logo" alt="company logo"> 
```

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,315 @@
...
+
+/* styling header */
+
+header {
+  background: linear-gradient(to left, #999999, #c5c5c5);
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,315 @@
...
+  height: 100px;
+  justify-content: space-evenly;
+  color: rgb(51, 51, 51);
+}
+
+.link-desktop-navbar {
+  text-decoration: none;
+  font-size: 20px;
+  color: rgb(58, 57, 57);
+    }
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
}
```

**Pull request closed without merging**